llm/llm_agents/gemini/gemini_mcp_client.py

```
@@ -0,0 +1,310 @@
+  "args": null
+}}
+
+Be selective and conservative with tool usage. Be concise. Only output valid JSON.
+Query: {query}
+"""
+    
+
+    async def is_mcp_server_ready(self, timeout=2):
+        try:
+            async with aiohttp.ClientSession() as session:
+                async with session.get(self.mcp_server_url, timeout=timeout) as resp:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)